### ISSUE TITLE
Class profile colors behave differently on "tool" and "demo" instances

### ISSUE THREAD

**Issue by jakubklimek** [mff-uk/dataspecer#796]
on Tool, the class profiles on canvas are colored by the profiled vocabulary
on Demo, the class profiles on canvas are colored by the profile
(also see #775 )
![image](https://github.com/user-attachments/assets/bd373c43-d567-40aa-b36d-30268c40fbf1)

**Comment by sstenchlak** [comment]
@skodapetr Can you have a look at it? It seems that data are corrupt.

https://tool.dataspecer.com/conceptual-model-editor/diagram?package-id=7f07c4ee-7d0f-471e-83ea-e5515c7c216c&view-id=7f07c4ee-7d0f-471e-83ea-e5515c7c216c%2F8f4ebd78-da6b-44e8-b93f-27b9bcc4b64a

**Comment by skodapetr** [comment]
The color is determined by the model which is stored in the visual model. 
As a result, the colors are effectively stored in the data.
Using two different versions of CME may cause.

**Comment by jakubklimek** [comment]
@skodapetr I think you are missing the point.

The difference is:
In one instance, the color of the class profile is determined by the color of the visual model of the profiled class.
In the other instance, the color of the class profile is determined by the color of the visual model of the class profile.

And the question is what caused this, whether it was intentional, or whether it is some kind of a bug.

**Comment by skodapetr** [comment]
A) Are both instances using the same version?
B) As this is stored in data, you need to re-create the visual entities.

**Comment by jakubklimek** [comment]
No, they are using different versions. And it seems there is a change between the versions. And the question is why, whether it was intended, or a bug,

**Comment by skodapetr** [comment]
May be part of the transition to Editor and introduction of Visual Model. 
I would use https://github.com/mff-uk/dataspecer/issues/775 to decide and later implement the desired state.
